### PR TITLE
feat(auth): can now login without accept

### LINKED ISSUE #203: Umiddelbar brukerregistrering uten godkjennelse / Forsinket godkjennelse
**Use-case:** Som besøkende ønsker jeg å kunne lage en bruker uten å måtte vente på å bli godkjent som medlem slik at jeg kan logge inn å ta i bruk siden.

Det er mye forvirring og venting knyttet til å måtte bli godkjent som medlem før man kan logge inn. Dette bør endres slik at man kan logge inn med en gang, uten tilgang til det et vanlig medlem har tilgang til. 

Hypotesen er at dette vil føre til en bedre brukeropplevelse og et bedre førsteinntrykk av både TIHLDE og nettsiden.

### PR DESCRIPTION
**Pull request by thomsen85**
## Proposed changes
You can now log in without being approved by admin, but you have limited accessibility.

Issue number: closes #203 

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] CHANGELOG.md has been updated. [Guide](https://tihlde.slab.com/posts/changelog-z8hybjom)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] The fixtures have been updated if needed (for migrations)
- [x] Pull request title follows [conventional commits](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) (`type(scope): description`)
- [x] Format and lint (`make format` and `make flake8`) was run locally without failures

### CODE REVIEW

File: app/content/models/registration.py [hunk 1]
```diff
@@ -59,7 +59,10 @@ def has_list_permission(cls, request):
 
     @staticmethod
     def has_write_permission(request):
-        return bool(request.user)
+        if request is None or request.user is None:
+            return False
+
+        return bool(request.user.is_TIHLDE_member)
```
Comment (Her0elt) [inline app/content/models/registration.py:65]: `return request.user and bool(request.user.is_TIHLDE_member)` kanskje

Comment (Her0elt) [inline app/content/models/registration.py:65]:
```suggestion
        return request.user and bool(request.user.is_TIHLDE_member)
```

Comment (Her0elt) [inline app/content/models/registration.py:65]:
```suggestion
        return request.user and bool(request.user.is_TIHLDE_member)
```

Comment (Her0elt) [inline app/content/models/registration.py:65]:
```suggestion
       return request.user and bool(request.user.is_TIHLDE_member)
```

Comment (thomsen85) [reply app/content/models/registration.py:65]: UUUUUUU clean :D

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,6 +13,9 @@
 ---
 
 ## Neste versjon
+- ⚡ **Pålogging** kan nå gjøres av brukere som ikke er godkjent, men har begrenset tilgang.
+
+## Versjon 1.1.5 (14.11.2021)
```
Comment (olros) [inline CHANGELOG.md:18]: Denne linjen må vekk
```suggestion
```

File: app/tests/conftest.py [hunk 3]
```diff
@@ -64,6 +64,7 @@ def token(user):
 def admin_user():
     user = UserFactory()
     add_user_to_group_with_name(user, AdminGroup.HS)
+    add_user_to_group_with_name(user, Groups.TIHLDE)
```
Comment (olros) [inline app/tests/conftest.py:67]: Mulig å ta inn member-fixture som param istedenfor å lage en user og legge til Groups.TIHLDE?

File: app/tests/content/test_registration_integration.py [hunk 4]
```diff
@@ -748,50 +748,50 @@ def test_delete_own_registration_as_member_when_no_users_on_wait_are_in_a_priori
 
 
 @pytest.mark.django_db
-def test_that_users_cannot_register_when_has_unanswered_evaluations(api_client, user):
+def test_that_users_cannot_register_when_has_unanswered_evaluations(api_client, member):
```
Comment (olros) [inline app/tests/content/test_registration_integration.py:751]: Oppdater tittel til members istedenfor users. Samme for de andre testene også
```suggestion
def test_that_members_cannot_register_when_has_unanswered_evaluations(api_client, member):
```

**Review by olros — CHANGES_REQUESTED** [review]

File: app/tests/content/test_registration_integration.py (hunk 4 above)
Comment (thomsen85) [reply app/tests/content/test_registration_integration.py]: Fikset

**Review by CJGutz — APPROVED** [review]
Bra jobba!

File: app/tests/content/test_short_link_integration.py [hunk 5]
```diff
@@ -49,7 +49,7 @@ def test_retrieve_as_anonymous_user(default_client, short_link):
     url = _get_short_link_url(short_link)
     response = default_client.get(url)
 
-    assert response.status_code == status.HTTP_200_OK
+    assert response.status_code == status.HTTP_403_FORBIDDEN
```
Comment (olros) [inline app/tests/content/test_short_link_integration.py:52]: Alle må kunne hente ut spesifikke short-links. Hvis ikke vil det ikke være mulig å dele/bruke dem.

File: app/content/models/short_link.py [hunk 6]
```diff
@@ -11,6 +11,7 @@ class ShortLink(BaseModel, BasePermissionModel):
     name = models.CharField(max_length=50, primary_key=True)
     user = models.ForeignKey(User, on_delete=models.CASCADE, related_name="short_links")
     url = models.URLField(max_length=600)
+    read_access = [Groups.TIHLDE]
```
Comment (olros) [inline app/content/models/short_link.py:14]: Kun medlemmer kan lage, alle må kunne lese spesifikke short-links
```suggestion
    write_access = [Groups.TIHLDE]
```

**Review by olros — CHANGES_REQUESTED** [review]

**Pull request closed without merging**